File: nelio-alves/algoritmo-e-logica-de-programacao/cpp/exercicios/par_impar.cpp
#include <iostream>

using namespace std;

int main() {
    int N, num;

    cout << "Quantos numeros voce vai digitar? ";
    cin >> N;

    for (int i = 0; i < N; i++) {
        cout << "Digite um numero: ";
        cin >> num;

        if (num == 0) {
            cout << "NULO" << endl;
        } else {
            if (num % 2 == 0) {
                cout << "PAR";
            } else {
                cout << "IMPAR";
            }

            if (num > 0) {
                cout << " POSITIVO" << endl;
            } else {
                cout << " NEGATIVO" << endl;
            }
        }
    }

	return 0;
}

File: nelio-alves/algoritmo-e-logica-de-programacao/python/exercicios/par_impar.py
N: int; num: int

N = int(input("Quantos numeros voce vai digitar? "))

for i in range(N):
	num = int(input("Digite um numero: "))

	if num == 0:
		print("NULO")
	else:
		if num % 2 == 0:
			print("PAR", end="")
		else:
			print("IMPAR", end="")
		if num > 0:
			print(" POSITIVO")
		else:
			print(" NEGATIVO")